app.js

```diff
@@ -8,9 +8,19 @@ if (process.env.mongoHost){
     //if we're launched from Docker
     mongoURL = 'mongodb://'+process.env.mongoHost+':27017/kort';
 }
-//the admin user is created upon launching the application for the first time
-const adminUser = "admin";  //optionally change this
-const adminPassword = "admin"; //set this to something different and secure
+
+//load in environment variables
+require('dotenv').config();
+
+//default admin user and password
+var adminUser = 'admin';
+var adminPassword = 'admin';
+if(process.env.adminUser && process.env.adminPassword){
+    adminUser = process.env.adminUser;
+    adminPassword = process.env.adminPassword;
+} else {
+    console.error("No admin user and/or password set, using default admin/admin");
+}
```

Comment:
Would recommend using `const` instead of `var`. I know it is not important, but good practice to do so ;)
What do you think about using warning instead of error level?

```suggestion
// Default admin user and password
const DEFAULT_ADMIN_USER = 'admin';
const DEFAULT_ADMIN_PASSWORD = 'admin';

const adminUser = process.env.ADMIN_USER || DEFAULT_ADMIN_USER;
const adminPassword = process.env.ADMIN_PASSWORD || DEFAULT_ADMIN_PASSWORD;

if (adminUser === DEFAULT_ADMIN_USER && adminPassword === DEFAULT_ADMIN_PASSWORD) {
    console.warn("No admin user and/or password set, using default admin/admin");
}
```

Reply:
Agree with Remi here